plug-ins/dns_spoof/dns_spoof.c

```diff
@@ -503,6 +524,7 @@ static int prepare_dns_reply(u_char *data, const char *name, int type, int *dns_
    struct rr_entry *rr;
    bool is_negative;
    int len;
+   u_int32 ttl = 3600, ttl_bige = 0x10e00000; /* 1 hour by default in case something goes wrong / 3600 big-endian */
```

Comment:
Please separate the declaration from the defenition as defined in C99 standard AFAIR.